fe2-android/app/src/test/java/com/github/dedis/popstellar/ui/pages/digitalcash/DigitalCashPageObject.java

```diff
@@ -1,55 +1,55 @@
-package com.github.dedis.popstellar.ui.pages.digitalcash;
-
-import androidx.annotation.IdRes;
-import androidx.test.espresso.ViewInteraction;
-
-import com.github.dedis.popstellar.R;
-
-import static androidx.test.espresso.Espresso.onView;
-import static androidx.test.espresso.matcher.ViewMatchers.withId;
-
-/**
- * Page object of {@Link DigitalCashMainActivity}
- *
- * <p>Creation : 20.05.2022
- */
-public class DigitalCashPageObject {
-
-  public static ViewInteraction fragmentContainer() {
-    return onView(withId(R.id.fragment_container_digital_cash));
-  }
-
-  public static ViewInteraction sendButton() {
-    return onView(withId(R.id.send_coin));
-  }
-
-  public static ViewInteraction receiveButton() {
-    return onView(withId(R.id.receive_coin));
-  }
-
-  public static ViewInteraction historyButton() {
-    return onView(withId(R.id.history_coin));
-  }
-
-  public static ViewInteraction homeButton() {
-    return onView(withId(R.id.home_coin));
-  }
-
-  public static ViewInteraction issueButton() {
-    return onView(withId(R.id.issue_coin));
-  }
-
-  public static ViewInteraction navBar() {
-    return onView(withId(R.id.digital_cash_nav_bar));
-  }
-
-  @IdRes
-  public static int digitalCashFragmentId() {
-    return R.id.fragment_digital_cash_home;
-  }
-
-  @IdRes
-  public static int digitalCashFragmentContainerId() {
-    return R.id.fragment_container_digital_cash;
-  }
-}
+package com.github.dedis.popstellar.ui.pages.digitalcash;
```

Comment:
how come the whole file changed? Did we switch newlines between LF and CRLF ?

Reply:
I have no idea, but I have noticed that some files are LF and some are CRLF and my Android Studio seems to adapt to the existing encoding file by file.
It would maybe be better to set a standard for the whole fe2 project and add the chosen format to the code convention document ?

Reply:
Agreed. My recommendation would be to use LF everywhere, and just ask Windows users to configure the relevant Git configurations (core.autocrlf , etc.) 
When there's no other PR open, I'd suggest you or @GabrielFleischer do a full `dos2unix` on the project codebase 😅